subsys/bluetooth/host/mesh/shell.c

```diff
@@ -779,6 +812,7 @@ static const struct shell_cmd mesh_commands[] = {
 	{ "reset", cmd_reset, NULL },
 	{ "input-num", cmd_input_num, "<number>" },
 	{ "input-str", cmd_input_str, "<string>" },
+	{ "provision", cmd_provision, "<NetKeyIndex> <addr> [IVIndex]" },
```

Comment:
Doesn't it make sense to allow the user to input key if it has one? I guess that would be convenient in case one has to reboot while already provisioned when there is no persistent storage.

Reply:
Yes, it makes sense. It's just something I've decided to do in later patches.